### PR TITLE
Enable-RemoteDesktop - Add RDP NLA Switch

### PR DESCRIPTION
**Pull request by cam1985**
RDP NLA (Network Level Authentication)  Switch Added as this was blocking RDP access when left enabled and no NLA configured.

### CODE REVIEW

File: Boxstarter.WinConfig/Enable-RemoteDesktop.ps1 [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+
+.LINK
+http://boxstarter.org
+
+#>
+
+    param(
+        [ValidateSet("On","Off")]
+        [String]
+		$NLA
```
Comment (mwrock) [inline Boxstarter.WinConfig/Enable-RemoteDesktop.ps1]: Can you make "On" the default. This is the default on most machines. II'm guessing your machine may have been previously set to off or this is GPO controlled but it seems weird that a GPO would set to off. Without a default, the validateset will raise an error if this is missing which will cause existing usages of this function to fail.

**Comment by mwrock** [comment]
Besides the comment above, this is great to have.

File: Boxstarter.WinConfig/Enable-RemoteDesktop.ps1 (hunk 1 above)
Comment (cam1985) [inline Boxstarter.WinConfig/Enable-RemoteDesktop.ps1]: Done

**Comment by mwrock** [comment]
thanks a bunch! FYI: I ended up changing the name of the parameter and made it a switch

**Pull request merged**